src/general/PauseMenu.cs

```diff
@@ -191,6 +225,28 @@ private void CloseSavePressed()
         SetActiveMenu("primary");
     }
 
+    private void ConfirmExit()
+    {
+        switch (exitType)
+        {
+            case ExitType.ReturnToMenu:
+            {
+                // Unpause the game
+                GetTree().Paused = false;
+
+                TransitionManager.Instance.AddScreenFade(ScreenFade.FadeType.FadeOut, 0.1f, false);
+                TransitionManager.Instance.StartTransitions(this, nameof(OnSwitchToMenu));
+                break;
```

Comment:
Very minor, but I think extracting the cases to a seperate method helps with readability a bit.